src/helpers/strings-es.ts

```diff
@@ -158,7 +159,16 @@
     "inbox-view-resources-title": "Recursos",
     "inbox-view-recently-completed-title": "Reciente",
     "inbox-view-recently-completed-empty-text": "No tienes elementos recientemente completados en tu bandeja de entrada.",
-    "inbox-view-history-button-text": "Ver historial de bandeja de entrada"
+    "inbox-view-history-button-text": "Ver historial de bandeja de entrada",
```

Comment:
I would populate all the Asthma strings just using ChatGPT

Reply:
Yep, that is what I have been doing.  I just have a bunch more to go.  😄